docs/content/guide/es/05_using-translate-directive.ngdoc

```diff
@@ -14,14 +14,14 @@ una directiva para traducir conenidos en la vista.
 La directiva `translate` puede utilizarse de muchas maneras distintas. Un
  uso general sería:
 ```
-<ANY translate>TRANSLATION_ID</ANY>
```

Comment:
actually, `ANY` is a well-known wildcard tag name in the AJS documentation. I would not translate this.